### ISSUE TITLE
Having issues getting python-pdfkit to work with newest release?

### ISSUE THREAD

**Issue by ghost** [JazzCore/python-pdfkit#25]
I'm trying to upgrade my `wkhtmltopdf` package to `0.12.2.1` and not having any luck.

It did work before the update. The reason I'm trying to update is to fix the _splitting_ of content between PDF pages.

I'm running - Ubuntu 64x  - Trusty
I started with getting the newest release of `wkhtmltopdf` http://sourceforge.net/projects/wkhtmltopdf/files/0.12.2.1/wkhtmltox-0.12.2.1_linux-trusty-amd64.deb/download?use_mirror=hivelocity

Out of the box I got `Configuration Error` I hard coded to simply get past this.

``` python
# -*- coding: utf-8 -*-
import subprocess
import sys

class Configuration(object):
    def __init__(self, wkhtmltopdf='', meta_tag_prefix='pdfkit-'):
        self.meta_tag_prefix = meta_tag_prefix

        self.wkhtmltopdf = wkhtmltopdf

        if not self.wkhtmltopdf:
            if sys.platform == 'win32':
                self.wkhtmltopdf = subprocess.Popen(                    ['where', 'wkhtmltopdf'], stdout=subprocess.PIPE).communicate()[0].strip()
            else:
                self.wkhtmltopdf = subprocess.Popen( ['which', 'wkhtmltopdf'], stdout=subprocess.PIPE).communicate()[0].strip()

        try:
            #with open(self.wkhtmltopdf) as f:
           with open("/usr/local/bin/wkhtmltopdf") as f:
                pass
        except IOError:
            raise IOError('No wkhtmltopdf executable found: "%s"\n'
                          'If this file exists please check that this process can '
                          'read it. Otherwise please install wkhtmltopdf - '
                          'https://github.com/JazzCore/python-pdfkit/wiki/Installing-wkhtmltopdf' % self.wkhtmltopdf)
```

Now I'm getting this error and do not know how to proceed.  I tried adding `shell=True` to the Popen above but that didn't work either.

```
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/rq/worker.py", line 543, in perform_job
    rv = job.perform()
  File "/usr/local/lib/python2.7/dist-packages/rq/job.py", line 490, in perform
    self._result = self.func(*self.args, **self.kwargs)
  File "/home/worker-1/Desktop/Dropbox/changeaddress/facts/jobs.py", line 864, in job_sharepdfs
    mymovepdf_link = build_mymovepdf(account_uuid, addresschange_uuid)
  File "/home/worker-1/Desktop/Dropbox/changeaddress/facts/jobs.py", line 608, in build_mymovepdf
    s3file = pdfkit.from_string( output.getvalue() , False )
  File "/usr/local/lib/python2.7/dist-packages/pdfkit/api.py", line 68, in from_string
    return r.to_pdf(output_path)
  File "/usr/local/lib/python2.7/dist-packages/pdfkit/pdfkit.py", line 93, in to_pdf
    stderr=subprocess.PIPE)
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1327, in _execute_child
    raise child_exception
OSError: [Errno 2] No such file or directory
```

Has anyone had any luck updating this?

**Comment by mweissen** [comment]
I have a similar problem.
There are 3 computers

A is a linux machine, Ubutu 14.04.1 with wkhtmltopdf 0.11.0 rc 1, pdfkit 0.4.1
B is a linux machine too, Ubuntu 14.04.01 with wkhtmltopdf 0.12.2.1, pdfkit 0.5
C is a Windows  8 computer with wkhtmltopdf 0.12.2.1, pdfkit 0.5.

Using wkhtmltopdf from the command line works without any problems on all three computers.

Computer A and pdfkit are working since a lot of months - very well!
Computer B is a new installation of wkhtmltopdf and pdfkit. No success after a lot of hours on this weekend. I had to change to version 0.11.0 rc1 and everything run like on computer A
Computer C: everything is ok.

I don't know is it a problem of wkhtmltopdf or a problem of pdfkit?
But anyway it would be nice to get a solution.

What I have found:
I have tried to convert a simple string "abc" to the file "/tmp/test.pdf".
The error message tells, that the command was "wkhtmltopdf - /tmp/test.pdf". Looks not very good - where has the "abc" gone?

Regards, Martin
